codebase/codebase.go

```diff
@@ -186,7 +216,7 @@ func (m *GormCodebaseRepository) Save(ctx context.Context, codebase *Codebase) (
 	if err := tx.Error; err != nil {
 		return nil, errors.NewInternalError(ctx, err)
 	}
-	log.Printf("updated codebase to %v\n", codebase)
+	log.Info(nil, nil, "updated codebase to %v\n", codebase)
```

Comment:
Use the ctx Context in the log stmt. That allows us to log identity and request id etc..